lib/wootric/test.js

```diff
@@ -0,0 +1,187 @@
+    analytics = new Analytics;
+    wootric = new Wootric(options);
+    analytics.use(plugin);
+    analytics.use(tester);
+    analytics.add(wootric);
+  });
+
+  afterEach(function(){
+    analytics.restore();
+    analytics.reset();
+    wootric.reset();
+    sandbox();
+  });
+
+  it('should have the right settings', function(){
+    // TODO: add any additional options or globals from the source file itself
+    // to this list, and they will automatically get tested against, like:
+    // integration('Wootric')
+    //   .global('__myIntegration')
+    //   .option('apiKey', '')
+    analytics.compare(Wootric, integration('Wootric')
+      .assumesPageview()
+        .option('account_token', '')
+        .global('wootricSettings')
+        .global('wootric_survey_immediately')
+        .global('wootric')
+     //   .tag('library', '<script src="http://wootric-staging.dev/assets/services/segmentioSnippet.js">')
+    )
+  });
+
+  describe('before loading', function(){
+    beforeEach(function(){
+      analytics.stub(wootric, 'load');
+      analytics.initialize();
+      analytics.page();
+    });
+
+    afterEach(function(){
+      wootric.reset();
+    });
+    it('should not have a wootric object', function(){
+      analytics.assert(!window.wootric);
+    })
+    describe('#initialize', function(){
+      it ('should setup the wootricSettings object', function(){
+        analytics.assert(window.wootricSettings);
+        analytics.assert(window.wootricSettings instanceof Object)
+        analytics.assert(window.wootricSettings.response_callback instanceof Function)
+      })
+    });
+
+    describe('should call #load', function(){
+      // TODO: test that .initialize() calls `.load()`
+      // you can remove this if it doesn't call `.load()`.
+    });
+  });
+
+  describe('loading', function(){
+    it('should load', function(done){
+      analytics.load(wootric, done);
+    });
+  });
+
+  describe('after loading', function(){
+    beforeEach(function(done){
+      analytics.once('ready', done);
+      analytics.initialize();
+      analytics.page();
+    });
+    it('should have created the global wootric object', function(){
+      analytics.assert(window.wootric instanceof Function);
+    })
+    describe('#identify', function(){
+      beforeEach(function(){
+        // TODO: stub the integration global api.
+        // For example:
+        //analytics.stub(window._wootric, 'identify');
+        analytics.spy(window._wootric, "identify");
+      });
+
+      it('should send an id', function(){
+        analytics.identify('id');
+        // TODO: assert that the id is sent.
+        // analytics.called(window.api.identify, 'id');
+      });
+
+      it('should send traits', function(){
+        analytics.identify({ trait: true });
+        //analytics.assert(window.wootricSettings.email, 'shawn@shawnmorgan.com');
+        // TODO: assert that the traits are sent.
+        analytics.called(window._wootric.identify);
```

Comment:
not seeing `window._wootric.identify` called in the identify call, a little confused..